include/kframework/evm.md

```diff
@@ -1135,81 +1135,61 @@ Operators that require access to the rest of the Ethereum network world-state ca
 
 ### Account Queries
 
-TODO: It's unclear what to do in the case of an account not existing for these operators.
-`BALANCE` is specified to push 0 in this case, but the others are not specified.
-For now, I assume that they instantiate an empty account and use the empty data.
-
```

Comment:
We pass the same conformance test-suite as the other implementations, and this behavior has been the same since 6 years ago when it was introduced: https://github.com/runtimeverification/evm-semantics/commit/29913b40a558f0066b8f277c850b6ea5a57f264e#diff-fbe2eecda4210ecd36a23352476c19f2ae691860a076a5e66a4886af8bcaf1d7R618

That comment was added when the semantics was very new and we were just beginning ot test it, before even RV took over the semantics.